src/isomorphic/ReactEntry.js

```diff
@@ -27,6 +27,12 @@ if (__DEV__) {
   cloneElement = ReactElementValidator.cloneElement;
 }
 
+const REACT_FRAGMENT_TYPE =
+  (typeof Symbol === 'function' &&
+    Symbol.for &&
+    Symbol.for('react.fragment')) ||
+  0xad9c;
```

Comment:
For elements we picked `0xeac7` because it kind of looks like `React`. What does this look like? :)

The other ones have used increasing numbers since.

https://github.com/facebook/react/blob/d9c1dbd61772f8f8ab0cdf389e70463d704c480b/packages/react-reconciler/src/isomorphic/ReactCoroutine.js#L23-L24

https://github.com/facebook/react/blob/d9c1dbd61772f8f8ab0cdf389e70463d704c480b/packages/react-reconciler/src/isomorphic/ReactPortal.js#L19

The next one would be `0xeacb`

Reply:
Wasn't aware of that, it looks like the number `44444` if you convert it to decimal. 😆